Update EIP-3540: Claify, improve and fill missing sections of EOF EIPs (#9662)
ethereum/EIPs: EIPS/eip-7480.md

## Security Considerations

> Gas cost of these new opcodes is similar to the legacy counterpart `CODECOPY`. These new instructions need to be carefully considered during implementation of EOF container validation algorithm.

Reviewer: counterpart is awkward here

```suggestion
Gas cost of these new opcodes is comparable to the legacy `CODECOPY` instruction. They must be carefully considered during the implementation of the EOF container validation algorithm.
```

> Gas cost of these new opcodes is similar to the legacy counterpart `CODECOPY`. These new instructions need to be carefully considered during implementation of EOF container validation algorithm.

Reviewer 2: ```suggestion
Gas cost of these new opcodes is similar to the legacy counterpart `CODECOPY`. These new instructions need to be carefully considered during implementation of the EOF container validation algorithm.
```
